### PR TITLE
Skin-only implementation of Evan Heritages design

### PR DESCRIPTION
**Pull request by fichl**
Evan Heritages design now fully implemented.
Skin only version, synced to main.

Screenshot_249

### CODE REVIEW

File: NeuralAmpModeler/config.h [hunk 1]
```diff
@@ -56,8 +56,16 @@
 
 #define ROBOTO_FN "Roboto-Regular.ttf"
 #define HELP_FN "help.svg"
-#define FILE_FN "file.svg"
+#define FILE_FN "SkinEHeritage_File.svg"
 #define FOLDER_FN "folder.svg"
-#define CLOSE_BUTTON_FN "close-button.svg"
+#define CLOSE_BUTTON_FN "SkinEHeritage_Close-button.svg"
```
Comment (hegedus90) [inline NeuralAmpModeler/config.h:61]: Are those old SVGs used anywhere else? If not, they can be removed as well.

File: NeuralAmpModeler/NeuralAmpModeler.cpp [hunk 2]
```diff
@@ -66,40 +66,41 @@ class IVUpdateableLabelControl : public IVLabelControl
 // Styles
 const IVColorSpec activeColorSpec{
   DEFAULT_BGCOLOR, // Background
-  PluginColors::NAM_1, // Foreground
-  PluginColors::NAM_2.WithOpacity(0.4f), // Pressed
-  PluginColors::NAM_3, // Frame
```
Comment (hegedus90) [inline NeuralAmpModeler/NeuralAmpModeler.cpp:71]: Are these old colour constants still used anywhere else? If not, the declarations could be removed.

Comment (fichl) [reply NeuralAmpModeler/NeuralAmpModeler.cpp:71]: NAM_1 is actually used, yes...
the other ones not i think. does it hurt to leave them ?

File: NeuralAmpModeler/config.h (hunk 1 above)
Comment (fichl) [reply NeuralAmpModeler/config.h:61]: you mean the actual .svg files inr /img folder ?
they shouldnt be needed anymore, right...

File: NeuralAmpModeler/NeuralAmpModeler.cpp (hunk 2 above)
Comment (sdatkinson) [reply NeuralAmpModeler/NeuralAmpModeler.cpp:71]: I don't mind leaving them. Nit-level

File: NeuralAmpModeler/NeuralAmpModeler.cpp [hunk 3]
```diff
@@ -188,14 +193,14 @@ NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo& info)
     const float ngAreaHeight = toggleHeight;
     const float ngAreaHalfWidth = 0.5f * noiseGateArea.W();
     const IRECT ngToggleArea = noiseGateArea.GetFromBottom(ngAreaHeight)
-                                 .GetTranslated(0.0f, ngAreaHeight + singleKnobPad)
+                                 .GetTranslated(0.0f, ngAreaHeight + singleKnobPad - 14.f)
```
Comment (sdatkinson) [inline NeuralAmpModeler/NeuralAmpModeler.cpp:196]: Nit: would prefer `14.f` as a variable

File: NeuralAmpModeler/NeuralAmpModeler.cpp [hunk 4]
```diff
@@ -206,33 +211,38 @@ NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo& info)
...
+    const IRECT irArea = content.GetFromBottom(fileHeight + 2.f).GetMidHPadded(fileWidth);
 
     // Areas for meters
-    const float meterHalfHeight = 0.5f * 250.0f;
+    const float meterHalfHeight = 0.5f * 385.0f;
     const IRECT inputMeterArea = inputKnobArea.GetFromLeft(allKnobsHalfPad)
                                    .GetMidHPadded(allKnobsHalfPad)
                                    .GetMidVPadded(meterHalfHeight)
-                                   .GetTranslated(-allKnobsPad, 0.0f);
+                                   .GetTranslated(-allKnobsPad - 18.f, 0.0f);
```
Comment (sdatkinson) [inline NeuralAmpModeler/NeuralAmpModeler.cpp:226]: Nit: `18.f`

File: NeuralAmpModeler/NeuralAmpModeler.cpp [hunk 5]
```diff
@@ -315,52 +325,67 @@ NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo& info)
...
         irArea.GetReducedFromLeft(iconWidth).GetReducedFromRight(iconWidth), this->mDefaultIRString.Get(),
-        style.WithDrawFrame(false).WithValueText(style.valueText.WithVAlign(EVAlign::Middle))),
+        style.WithDrawFrame(false).WithValueText(style.valueText.WithSize(16.f).WithVAlign(EVAlign::Middle))),
       kCtrlTagIRName);
 
     // NG toggle
-    IVSlideSwitchControl* noiseGateSlider = new IVSlideSwitchControl(ngToggleArea, kNoiseGateActive, "Gate", style,
-                                                                     true, // valueInButton
-                                                                     EDirection::Horizontal);
+    // underlaying background in theme color for ON state
```
Comment (sdatkinson) [inline NeuralAmpModeler/NeuralAmpModeler.cpp:353]: Clever :)

**Review by sdatkinson — APPROVED** [review]
Rock & roll 🤘🏻

**Comment by olilarkin** [comment]
This is a nice new skin! however there are some things in the PR that got merged that should be cleaned up (duplicate controls etc). I will do a PR to simplify the grey-ing out of controls in which i will remove the doubled up controls

**Comment by olilarkin** [comment]
Aha... ok i see what you are doing with the doubled controls. This would be cleaner with a dedicated control - i'll add

**Comment by olilarkin** [comment]
Also for future reference there is some incorrect usage of the control tag and "group" parameters to the IGraphics::AttachControl() calls for the new IBKnobRotaterControls. Will fix!

**Pull request merged**